### PR TITLE
Domain admin context

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/30643

## Description
Changed Whois Domain.Admin to correspond to context standards.
Changed Domain default mapper to correspond to context standards.

### CODE REVIEW

**Comment by Itay4** [comment]
@ShahafBenYakir please add release notes which mention the breaking change

File: Packs/Whois/Integrations/Whois/Whois.py [hunk 1]
```diff
@@ -8331,7 +8331,8 @@ def create_outputs(whois_result, domain, query=None):
         if 'admin' in contacts and contacts['admin'] is not None:
             md['Administrator'] = contacts['admin']
             ec['Administrator'] = contacts['admin']
-            standard_ec['Admin'] = contacts['admin']
+            standard_ec['Admin'] = {key.capitalize(): val for key, val in contacts['admin'].items()}
+            print(standard_ec['Admin'])
```
Comment (Itay4) [inline Packs/Whois/Integrations/Whois/Whois.py:8335]: remove print

File: Packs/Whois/Integrations/Whois/Whois.py [hunk 2]
```diff
@@ -8331,7 +8331,8 @@ def create_outputs(whois_result, domain, query=None):
         if 'admin' in contacts and contacts['admin'] is not None:
             md['Administrator'] = contacts['admin']
             ec['Administrator'] = contacts['admin']
-            standard_ec['Admin'] = contacts['admin']
+            standard_ec['Admin'] = {key.capitalize(): val for key, val in contacts['admin'].items()}
```
Comment (Itay4) [inline Packs/Whois/Integrations/Whois/Whois.py:8334]: output to both paths

**Review by Itay4 — APPROVED** [review]

**Comment by kirbles19** [comment]
@ShahafBenYakir Done

**Comment by ShahafBenYakir** [comment]
Force merging.
The issue is with jsonwhois service seems to be unavailable.

File: Packs/Whois/Integrations/Whois/Whois.py [hunk 3]
```diff
@@ -8331,6 +8331,7 @@ def create_outputs(whois_result, domain, query=None):
         if 'admin' in contacts and contacts['admin'] is not None:
             md['Administrator'] = contacts['admin']
             ec['Administrator'] = contacts['admin']
+            standard_ec['Admin'] = {key.capitalize(): val for key, val in contacts['admin'].items()}
             standard_ec['Admin'] = contacts['admin']
```
Comment (ShahafBenYakir) [inline Packs/Whois/Integrations/Whois/Whois.py:8335]: @Itay4 I think this is an issue....

**Pull request merged**